lib/records/RecFile.cc

```diff
@@ -88,6 +88,26 @@ RecFileRead(RecHandle h_file, char *buf, int size, int *bytes_read)
   return REC_ERR_OKAY;
 }
 
+//-------------------------------------------------------------------------
+// RecSnapFileWrite
+//-------------------------------------------------------------------------
+
+int
+RecSnapFileWrite(RecHandle h_file, char *buf, int size, int *bytes_written)
+{
+  // First write the version byes for snap file
+  char version_msg[5] = {'V', PACKAGE_VERSION[0], PACKAGE_VERSION[2], PACKAGE_VERSION[4], '\0'};
```

Comment:
Do not have magic constants like "5" floating around the code. `std::array` would be a better choice.
```
std::array<char, 5> VERSION_HDR { 'V', ...};
if (::write(h_file, VERSION_HDR.data(), VERSION_HDR.size()) < 0 ) { ...}
```

Reply:
Agree.